WordPress/gutenberg
# Make template part and reusable block creation language consistent

## What?
This PR updates menu items that enable reusable block or template part creation to use consistent language.

## Why?
Consistent language surrounding similar features strengthens trust in the overall UX.

## How?
The following text strings are updated:

* "Add to Reusable blocks" becomes "Create Reusable block"
* "Make template part" becomes "Create Template part"

I chose "Create" because it is short, and aligns with both the modal titles in the creation flow, and the confirmation snackbars post creation, e.g. "Template part created". But there are other options we might consider – let's discuss!

In addition to the language update, I appended an icon to the "Create Template part" menu item.

## Testing Instructions
In the site editor select a block, open the more options menu and observe the two menu items:

<img width="628" alt="Screenshot 2022-08-05 at 16 10 23" src="https://user-images.githubusercontent.com/846565/183107902-782f969b-17d5-4886-aee6-e0d849d040bb.png">
